Add chemistry sprites. Move existing ones into RSIs

Moneyl opened the pull request:
There are already a few chemistry sprites that are sitting loose in folders. This adds chemistry sprites from CEV-Eris that'll be needed in upcoming PRs and moves existing ones into their relevant RSIs. I've made this separate from any other chemistry additions to keep things simple.

PJB3005 commented:
You should separate out the powered/lit layers, like I did with stuff like computers and airlocks

Moneyl commented:
@PJB3005 They should be split up properly now. I split the screens and the machine sprites into their own files and updated meta.json with the new info. 

![explorer_Kxy64BhoT1](https://user-images.githubusercontent.com/8206401/68064882-fdff4f00-fcf7-11e9-8670-3014e22c5467.png)
